### ISSUE TITLE
Request : Syslog function | more logging

### ISSUE THREAD

**Issue by floppy69** [pseymour/MakeMeAdmin#122]
Hello,

Thanks for all the work you've done here

Can it be added in syslogs traps which soft/command/ was used with admin right ?
Can it also be added a registry value for asking or not user to tell what he wanted admin right ?

regard and thanks in advance even if the answer is no

**Comment by hirnerstefan** [comment]
I can only agree with that, a really very successful tool. The only thing I'm really missing here is that the administrative entries (which are already written to the event log) are passed to the syslog server (currently only the additions and removals by the user are logged)

Thanks again for your work and all the best

**Comment by teorouge** [comment]
Yes, more logging would be so useful, hope you can implement that... maybe even add a user-facing textbox where users write the reason they're asking admin credentials for, which gets logged too. Thanks for this nice piece of software!

**Comment by JosephGonzalez-NYPL** [comment]
We ran into this exact issue. After some digging, we found why — the elevation/removal events (0, 1, 9) go through ApplicationLog.WriteEvent() which has the syslog send logic. But the elevated process events (101) go through a completely different path — LoggingProvider which extends EventSource and only writes to the local Event Log. Syslog is never called for those.

So the process logging works great locally (we can see full command lines, PIDs, elevation types in the MakeMeAdmin Event Log), it just never makes it to the syslog server. The other events come through fine over TCP, so connectivity isn't the issue.

Would love to see this addressed — being able to see what users actually ran while elevated is huge for auditing. Happy to help test if a fix is in the works!
